File: Weather.Business/V1/Logic/CMS/NewsCategory/NewsCategoryModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Weather.Business.V1
{
    public class NewsCategoryFilterModel : BaseQueryFilterModel
    {
        public Guid? NewsCategoryId { get; set; }
        public string Type { get; set; }
    }

    public class NewsCategoryCreateRequestModel
    {
        //public Guid Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Type { get; set; }
        public int Order { get; set; }

        public Guid CreatedByUserId { get; set; }
    }

    public class NewsCategoryUpdateRequestModel
    {
        public Guid NewsCategoryId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Type { get; set; }
        public int Order { get; set; }

        public Guid LastEditedByUserId { get; set; }
    }

    public class NewsCategoryDeleteResponseModel : BaseDeleteResponseModel
    {
    }
}

File: _VienKhiTuongThuyVan/KTTVPortal/src/app/CMS/services/newscategory/newscategory.model.ts
import { BaseQueryFilter } from 'src/app/_layout/utils/common-classes';

export class NewsCategory {
    NewsCategoryId: string;
    Name:string;
    Description:string;
    Type: string = null;
    Order: number;
    CreatedOnDate: string;
    LastEditedOnDate: string;
    CreatedByUserId: string;
    LastEditedByUserId: string;  
  }
  export class NewsCategoryFilter extends BaseQueryFilter{
    NewsCategoryId: string = null;
    Type: string = null;
  }
  export class NewsCategoryCreateRequestModel {
    Name: string;
    Description: string;
    Type: string;
    Order: number;
    CreatedByUserId: string = null;
  }
  export class NewsCategoryUpdateRequestModel {
    NewsCategoryId: string = null;
    Name: string;
    Description: string;
    Type: string;
    Order: number;
    LastEditedByUserId: string = null;
  }